src/index.html

```diff
@@ -23,13 +23,54 @@ <h2 class="logo-text">Bookoe</h2>
         </div>
     </header>
     <main>
-        
+
     </main>
     <footer>
-
+        <div class="row upper-footer">
+            <div class="col-3">
+                <div class="logo">
+                    <h2 class="logo-text">Bookoe</h2>
+                    <p class="logo-disc">Book store Website</p>
+                </div><br><br>
+                <p>
+                    “The pages are still blank, but there is a miraculous feeling of the words being there, written in
+                    invisible ink and clamoring to become visible”
+                    ― Vladimir Nabokov
+                </p> <br><br>
+                <h3>Follow us</h3><br>
+                <ul id="social-icons">
+                    <li><a href="#"><i class="fa-brands fa-facebook"></i></a></li>
+                    <li><a href="#"><i class="fa-brands fa-youtube"></i></a></li>
+                    <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+                    <li><a href="#"><i class="fa-brands fa-linkedin"></i></a></li>
+                    <li><a href="#"><i class="fa-brands fa-instagram-square"></i></a></li>
+                </ul>
+            </div>
+            <div class="col-3">
+                <h3>Book Categories</h3>
+                <ul id="book-categories">
+                    <li><a href="#">Action</a></li>
+                    <li><a href="#">Advanture</a></li>
+                    <li><a href="#">Commedy </a></li>
+                    <li><a href="#">Crime </a></li>
+                    <li><a href="#">Drama</a></li>
+                    <li><a href="#">Horror</a></li>
+                </ul>
+            </div>
+            <div class="col-4">
+                <h3>Contact Us</h3>
+                <div id="contacts">
+                    <span><i id="contact-icons" class="fa-solid fa-phone"></i> +254719584717 / +251925106658 </span>
+                    <span><i id="contact-icons" class="fa-solid fa-envelope"></i>omayioben@gmail.com /
```

Comment:
[Optional] kindly you can style the icons of the phone and email color to the primary color so that the page color usage looks consistent.